roles/lib_openshift/library/oc_atomic_container.py

```diff
@@ -191,9 +191,15 @@ def main():
         )
 
     # Verify that the platform supports atomic command
-    rc, _, err = module.run_command('atomic -v', check_rc=False)
+    rc, version_out, err = module.run_command('atomic -v', check_rc=False)
     if rc != 0:
         module.fail_json(msg="Error in running atomic command", err=err)
+    # This module requires atomic version 1.17.2 or later
+    version_out = version_out.replace('\n', '')
+    if version_out < '1.17.2':
```

Comment:
should we use distutils.version to compare versions?  atomic won't probably get to 1.17.10, but in that case the string comparison will fail.